.cirrus.yml

```diff
@@ -53,8 +60,6 @@ qa_plugin_task:
       - echo $CIRRUS_OS
       - cat **/*.gradle* **/gradle*.properties || true
     populate_script: mkdir -p ${GRADLE_USER_HOME}/caches
```

Comment:
In sonar-go we're following the sonar-kotlin approach to create a template for the gradle cache; see [this PR](https://github.com/SonarSource/sonar-go/pull/2/files )